src/components/Burst.js

```diff
@@ -38,24 +37,34 @@ class Burst extends Component {
   }
 
   render() {
-    const { idx, min, max, step, errors } = this.state;
-    const { expanded } = this.props;
+    const { min, max, step, errors } = this.state;
+    const { expanded, burstSliders } = this.props;
 
     if (!expanded) return <div className="Burst" />;
 
     return (
       <div className={classNames('Burst', { 'Burst-expanded': expanded })}>
-        <div>Slider Index</div>
-        <input
-          className={classNames('Burst-input', {
+        <div>Slider</div>
+        <select
+          className={classNames('Burst-dropdown', {
             'Burst-input-error': !!errors.idx
           })}
-          type="number"
           name="idx"
           aria-label="slider index"
-          value={isNaN(idx) ? '' : idx}
           onChange={this.handleInputUpdate}
-        />
+        >
+          <option value={null} defaultValue>
+            {burstSliders.length ? 'Choose Slider' : 'No Sliders'}
```

Comment:
can you change the 'Choose Slider' text to something shorter? the text is getting cut off:

<img width="140" alt="Screen Shot 2019-06-25 at 11 38 27 AM" src="https://user-images.githubusercontent.com/3916770/60124140-d55dbc00-973d-11e9-9f70-6164efd6c75b.png">

Reply:
Changed to `Pick Slider`, now it fits